Library/Formula/imply.rb

```diff
@@ -0,0 +1,27 @@
+class Imply < Formula
+  desc "Exploratory Analytics Powered By Druid"
```

Comment:
I'm not really sure what this actually means? Can we try to get a better description?